### PR TITLE
Fix namespace name input empty error

### PR DESCRIPTION
**Pull request by SehKeat**
Error text will appear if name input is empty.

### CODE REVIEW

**Comment by sleepyOwl14** [comment]
onClick display empty error not so correct

**Comment by SehKeat** [comment]
noted

**Comment by SehKeat** [comment]
changed to work similar with how password showing the error

File: src/modules/services/submodule/namespaces/views/ViewServicesNamespaceCreate.vue [hunk 1]
```diff
@@ -522,6 +522,13 @@ export default {
       }
     }
 
+    const checkEmptyNamespace = async () =>{
+      if (namespaceName.value.length == 0){
+        showNamespaceNameError.value = true;
+        namespaceErrorMessage.value = t('namespace.validName');
+      }
+    }
+
```
[the remark this answers is not in the document]
Comment (SehKeat) [reply src/modules/services/submodule/namespaces/views/ViewServicesNamespaceCreate.vue:534]: noted, else statement added

[the remark this answers is not in the document]
Comment (anakin001) [reply src/modules/services/submodule/namespaces/views/ViewServicesNamespaceCreate.vue:534]: your else statement do nothing. should be showNamespaceError.value = false?

[the remark this answers is not in the document]
Comment (SehKeat) [reply src/modules/services/submodule/namespaces/views/ViewServicesNamespaceCreate.vue:534]: if showNamespaceError.value = false added, the error test appear when out focus while the input is empty then if the length become 1 which is still less the required amount(min length =2), the error test from the empty input would disappear for 1 second then appear again. if not added, the error text will only disappear when meet the requirement. Should I  showNamespaceError.value = false?

**Comment by sleepyOwl14** [comment]
please use blur event instead of focusout

**Pull request closed without merging**